python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -206,6 +206,7 @@ def __call__(self):
 @pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
 @pytest.mark.parametrize("smoothing_factor", [1, 0.2])
 @pytest.mark.parametrize("use_upscale_downscale_config", [True, False])
+@mock.patch("ray.serve._private.router.HANDLE_METRIC_PUSH_INTERVAL_S", 1)
```

Comment:
In this test (for `RAY_SERVE_COLLECT_AUTOSCALING_METRICS_ON_HANDLE=0`) when the deployment scales back down to 0, the last metric report pushed from the handle was a non-zero number (because the push interval is set to 10 seconds). So for the next 10 seconds the # replicas keeps oscillating between 0 and 1 because of the outdated metric from the handle. I'm not sure why we never ran into this before, but this seems like a totally reasonable scenario so I set the handle push interval lower to avoid making the test wait longer.